plugins/modules/fabric_virtual_networks_workflow_manager.py

```diff
@@ -0,0 +1,2950 @@
+                                  "ip_directed_broadcast", "intra_subnet_routing_enabled", "multiple_ip_to_mac_addresses"]
+            for key, value in anycast_mapping.items():
+                playbook_param = anycast.get(key)
+
+                if playbook_param is not None:
+                    anycast_payload[value] = playbook_param
+                elif playbook_param is None and key in params_enable_list:
+                    anycast_payload[value] = False
+
+        if anycast.get("auto_generate_vlan_name") is True or anycast_payload.get("isCriticalPool") is True:
+            anycast_payload.pop("vlanName", None)
+            anycast_payload.pop("vlanId", None)
+            anycast_payload["autoGenerateVlanName"] = True
+
+        return anycast_payload
+
+    def is_gateway_needs_update(self, anycast, anycast_details_in_ccc):
+        """
+        Checks if the Anycast Gateway configuration needs to be updated based on provided parameters.
+        Args:
+            self (object): An instance of a class used for interacting with Cisco Catalyst Center.
+            anycast (dict): A dictionary containing the current configuration parameters for the Anycast Gateway.
+            anycast_details_in_ccc (dict): A dictionary containing the current Anycast Gateway configuration details
+                                            from the Cisco Catalyst Center (CCC) to compare against.
+        Returns:
+            bool: True if any of the relevant parameters differ between the provided `anycast` configuration and the
+                corresponding parameters in `anycast_details_in_ccc`, indicating that an update is needed.
+                Returns False if no differences are found.
+        Description:
+            This function compares specific configuration parameters of an Anycast Gateway between the provided
+            configuration (`anycast`) and the existing configuration in the Cisco Catalyst Center.
+            If any differences are detected, it logs an informational message and returns True. If no discrepancies
+            are found, the function returns False, indicating that the current configuration is up to date.
+        """
+
+        update_param_to_check = ["tcp_mss_adjustment", "traffic_type", "security_group_name", "layer2_flooding_enabled", "fabric_enabled_wireless"
+                                 , "ip_directed_broadcast", "multiple_ip_to_mac_addresses", "supplicant_based_extended_node_onboarding"
+                                 , "group_policy_enforcement_enabled"]
+        vn_name = anycast.get("vn_name")
+        anycast_mapping = self.get_anycast_gateway_mapping(vn_name)
+
+        if vn_name == "INFRA_VN":
+            params_to_remove = ["security_group_name", "layer2_flooding_enabled", "fabric_enabled_wireless",
+                                "ip_directed_broadcast", "multiple_ip_to_mac_addresses"]
+            for param in params_to_remove:
```

Comment:
for param in params_to_remove:
            if param in update_param_to_check:
                update_param_to_check.remove(param)
                self.log("Removing parameter '{0}' from update check for INFRA_VN.".format(param), "DEBUG")